makefiles/Makefile.devhub-content

```diff
@@ -45,51 +45,20 @@ next-gen-html: next-gen-html-new
 		exit 0; \
 	fi
 	rsync -az --exclude '.git' ${REPO_DIR}/../../snooty-devhub ${REPO_DIR}; 
-	cp ${REPO_DIR}/.env.production ${REPO_DIR}/snooty-devhub; 
+	cp ${REPO_DIR}/.env.production ${REPO_DIR}/snooty-devhub;
 	cd snooty-devhub; \
 	echo "GATSBY_SITE=${PROJECT}" >> .env.production; \
 	echo "COMMIT_HASH=${COMMIT_HASH}" >> .env.production; \
 	echo "STRAPI_URL=${STRAPI_URL}" >> .env.production; \
+	# If prod --> set PATH_PREFIX build arg
```

Comment:
We would like for this to be not supplied if a staging job (default) and /developer if a prod job please